store/store.go

```diff
@@ -1391,3 +1409,12 @@ func (s *Store) PaymentMethods(user *auth.UserState) (*PaymentInformation, error
 		return nil, fmt.Errorf("cannot get payment methods: unexpected HTTP code %d%s", resp.StatusCode, details)
 	}
 }
+
+func (s *Store) listRefreshFields() []string {
+	fields := append([]string{}, s.detailFields...)
```

Comment:
you don't need to copy all the fields: you can do
```golang
fields := s.detailFields
if os.Getenv("...") {
    fields = append(fields, "deltas")
}
return fields
```

and fields will be what you expect